src/test/java/com/iexec/worker/compute/pre/PreComputeServiceTests.java

```diff
@@ -204,8 +201,8 @@ void shouldRunTeePreComputeAndPrepareInputDataWhenOnlyDatasetIsPresent() throws
 
     @Test
     void shouldRunTeePreComputeAndPrepareInputDataWhenOnlyInputFilesArePresent() throws TeeSessionGenerationException {
-        taskDescription.setDatasetAddress("");
-        taskDescription.setInputFiles(List.of("input-file1"));
+        final TaskDescription taskDescription = taskDescriptionBuilder
+                .datasetAddress("").inputFiles(List.of("input-file1")).build();
```

Comment:
I find it clearer on multiple lines.
```suggestion
        final TaskDescription taskDescription = taskDescriptionBuilder
                .datasetAddress("")
                .inputFiles(List.of("input-file1"))
                .build();
```

Reply:
done in d6b8477